### PR TITLE
Create alert

### PR DESCRIPTION
**Pull request by dvantetering**

### CODE REVIEW

**Comment by ric-jon** [comment]
Looks good, not a large change

File: src/main/java/core/graph/Graph.java [hunk 1]
```diff
@@ -37,7 +37,7 @@
 
     private ArrayList<String> allGenomes = new ArrayList<>();
 
-    private boolean filtering;
+    //private boolean filtering;
```
Comment (arbreurkes) [inline src/main/java/core/graph/Graph.java]: this can be removed too.

File: src/main/java/core/graph/Graph.java [hunk 2]
```diff
@@ -353,7 +345,6 @@ public void addCell(HashMap<Integer, Node> nodeMap, ArrayList<String> ref, Model
                 n.setLinks(new ArrayList<>());
             }
         }
-        System.out.println("finished topological sort");
         //return the sorted list
```
Comment (arbreurkes) [inline src/main/java/core/graph/Graph.java]: all these comments state the obvious

Comment (dvantetering) [inline src/main/java/core/graph/Graph.java]: Fixed!

File: src/main/java/core/graph/Graph.java (hunk 1 above)
Comment (dvantetering) [inline src/main/java/core/graph/Graph.java]: Fixed!

**Pull request merged**